jupyterhub/apihandlers/users.py

```diff
@@ -18,6 +18,43 @@ def get(self):
         users = self.db.query(orm.User)
         data = [ self.user_model(u) for u in users ]
         self.write(json.dumps(data))
+    
+    @admin_only
+    @gen.coroutine
+    def post(self):
+        data = self.get_json_body()
+        print(data)
+        if not data or not isinstance(data, dict) or not data.get('usernames'):
+            raise web.HTTPError(400, "Must specify at least one user to create")
+        
+        usernames = data.pop('usernames')
+        self._check_user_model(data)
+        
+        for name in usernames:
+            user = self.find_user(name)
+            if user is not None:
```

Comment:
As an alternative, it could collect all already existing users and provide a warning at the end about those users not having been re-created... Just a thought.